app/views/layouts/application.html.haml

```diff
@@ -32,10 +32,8 @@
     <![endif]-->
 
     = jquery_include_tag
-
-    - unless @landing_page
```

Comment:
Still any other uses of that variable left? Else I'd drop it from the controller too, including the application JS will likely have some potential to break custom landing pages anyway.